boilerplate.js

```diff
@@ -59,6 +60,13 @@ async function install(context) {
   ]
   filesToRemove.map(filesystem.remove)
 
+  const askAboutDetox = !parameters.options.detox
```

Comment:
```suggestion
  const askAboutDetox = parameters.options.detox === undefined
```

You'll want to check for `undefined` explicitely, because if they pass in `--no-detox` and are still prompted, it'll be annoying.

Reply:
Apparently, `--no-detox` results in a string of 'false', rather than a boolean, which took me a while to figure out, so it works as-is (I did test it :)), but I can change to undefined to be safe.